src/containers/Signin/Signin.tsx

```diff
@@ -62,15 +77,16 @@ const Login: FC<IAppProps> = props => {
           type="submit"
           variant="contained"
           color="primary"
-          isLoading={isLoading}
+          isLoading={signingIn}
```

Comment:
would prefer if you do the renaming at the variable declaration.

Reply:
I'm sorry didn't quite get this one

Reply:
just declare your boolean as `isLoading` instead of `signingIn`